README.md

```diff
@@ -118,100 +114,35 @@
 
 # 1. Introduction
 
-BCF is a format for managing issues on a BIM project. The BCF-API supports the exchange of BCF issues between software applications via a [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) web interface, which means that data is exchanged via Url-encoded query parameters and Json bodies over the Http protocol. Every resource described in this API has a corresponding Json schema (schema version draft-03).
-Url schemas in this readme are relational to the BCF base servers Url if no absolute values are provided.
+BCF is an open standard for managing issues on a BIM project. The BCF-API supports the exchange of BCF issues between software applications via a [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) web interface, which means that data is exchanged via HTTP query parameters and Json bodies. Every resource described in this API has a corresponding Json schema (schema version draft-03).
+Url schemas in this README are relative to the BCF servers' base Url unless absolute values are provided.
 
 For security reasons, all API Http traffic should be sent via TLS/SSL over Https connection. Clients and Servers should both enforce secure connections and disallow unsafe connections.
```

Comment:
This paragraph about http traffic I think is better suited for foundations. 

Another comment, do we propose any miminum version of TLS? From a security perspective > 1.2 is recommended these days (can this be included for the foundations 1.0 release).

Reply:
Resolved in the group to leave it here and also make a similar note in Foundations. 

Also add a recommendation in foundation around the TLS protocol.

Reply:
https://github.com/buildingSMART/opencde-foundation-API/pull/14